src/Intro.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import key from './api/api.js';
+
+function Intro () {
+
+  let city = '';
+
+  function inputChange (e) {
```

Comment:
I would use es6 syntax for functions that aren't React components - but that's up to you

Reply:
Updated my function, using const and arrow functions